packages/components/pie-button/src/index.ts

```diff
@@ -43,6 +58,97 @@ export class PieButton extends LitElement implements ButtonProps {
     @property({ type: Boolean })
     public isFullWidth = false;
 
+    @property({ type: String })
+    public name?: string;
+
+    @property({ type: String })
+    public value?: string;
+
+    @property({ attribute: 'formaction' })
+    public formAction: ButtonProps['formAction'];
```

Comment:
Just curious if there was a reason why you went for camelCase prop names instead of aligning with the lowercase attribute? If I understand correctly we could either do what you've done, or go with the lowercase name and use `reflect: true`?

Reply:
I thought this would be a bit more readable to reference in our code but I'm happy to reflect if you'd prefer? I am unsure if either approach has a performance benefit